Quemepasa/Diplom_3
# Add UI autotests

UI autotests have been added to the project:
- Successful registration
- An error for an incorrect password
- Login using the "Log in to account" button on the main page 
- Login using the "Personal Account" button 
- Login using the button in the registration form 
- Login using the button in the password recovery form
- Transfer to personal account
- Switching from personal account to the constructor using constructor button
- Switching from personal account to the constructor using Stellar Burgers logo
- Log out of account
- Check that the transitions to the section "Buns"
- Check that the transitions to the section "Sauces"
- Check that the transitions to the section "Fillings"
